Formula/bork.rb

```diff
@@ -1,8 +1,8 @@
 class Bork < Formula
   desc "Bash DSL for config management"
   homepage "https://github.com/mattly/bork"
-  url "https://github.com/mattly/bork/archive/v0.10.0.tar.gz"
-  sha256 "c06433ee2879b89e116a940c1549089e41e6e07da9bc1241a305bc72c14a9494"
+  url "https://github.com/mattly/bork/archive/0.11.0.tar.gz"
```

Comment:
decided to drop the v?

Reply:
eh, more of an omission when I tagged the release than anything else. Since you can't delete tags/releases from GH I decided to just go with it.

Reply:
You can. `git push origin :refs/tags/0.11.0`. Not that I'm complaining.

Reply:
that indeed worked. It didn't used to. I'll update the PR here in a few.

Reply:
and... done